frontend/package.json

```diff
@@ -31,6 +32,7 @@
     "@types/react": "^19.1.5",
     "@types/react-dom": "^19.1.5",
     "@vitejs/plugin-react": "^4.5.0",
+    "ajv-cli": "^5.0.0",
```

Comment:
Hmm, I'm getting a "high security vulnerability" from this dependency (or rather a dependency of this dependency):

<img width="996" alt="Screenshot 2025-05-28 at 11 24 48" src="https://github.com/user-attachments/assets/0203eafc-2a35-46f0-80c8-1b9853b9ccc4" />

Reply:
This is the vulnerability:
https://github.com/advisories/GHSA-8gh8-hqwg-xf34

This is an open PR to `ajv-cli` to fix it and bump the dependency, but it has been unaddressed in 2 years lol:
https://github.com/ajv-validator/ajv-cli/issues/225

Reply:
@AlexAxthelm any ideas? 
It seems that `ajv` introduces a vulnerability that doesn't seem like it's going to be addressed by the maintainers

Reply:
@cjyetman it seems like this fork is current/ up-to-date/ maintained:
https://github.com/jirutka/ajv-cli

Can be added as a dev dependency with:
`npm add --save-dev @jirutka/ajv-cli`

Reply:
And looks like if we want to use that forked version of `ajv` instead, the syntax would need to be updated slightly from:
`ajv -s pbtar_schema.json -d src/data/scenarios_metadata.json`
to:
`ajv validate -s pbtar_schema.json src/data/scenarios_metadata.json`